recipes/date/all/patches/cmake-3.0.0.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
```

Comment:
what about upstreaming this patch?

Reply:
I'll try but upstream development is frozen:

> This will likely be the final release of this library. From here you should migrate to C++20 as vendors begin to ship this library under namespace std::chrono.
> ...
> The CMake support of this library is chaotic (to be kind). Personally I don't use it and instead follow the installation instructions found here: https://howardhinnant.github.io/date/tz.html#Installation (needed only for tz.h).

Reply:
Done. Patch accepted in upstream. Next version of the recipe will be patch-free.